lib/github-status/support/git.rb

```diff
@@ -17,6 +17,8 @@ def git
 
       Contract None => String
       def sha
+        @sha ||= (File.read "#{workdir}/#{path}").chomp
+      rescue Errno::EISDIR
```

Comment:
Would `File.directory?` or `Dir.exist?` be clearer than rescuing `Errno::EISDIR`?

Reply:
... or `File.file?`, maybe?

Reply:
Should be fixed in 1d47d970d119706c2b5bb1e473897ec139f37b10

Reply:
👍